### ISSUE TITLE
MediaTypes.`image/svg+xml` is binary but should be text

### ISSUE THREAD

**Issue by nwtgck** [akka/akka-http#2570]
Hi, developers. I'm very new to create an issue in Akka.

I think ``` MediaTypes.`image/svg+xml` ``` is currently binary. But SVG is just XML, so it should be a text format.

**Comment by raboof** [comment]
I think I agree (like `application/xml`, see also https://www.w3.org/TR/SVG/mimereg.html). Unfortunately changing it now would be a binary-incompatible change, and the fact that it now fits into the compression infrastructure is nice. Does this lead to any particular problems for you?

**Comment by nwtgck** [comment]
> Does this lead to any particular problems for you?

@raboof No. I have no problem. I just convert SVG string to `ByteString` and use it as binary. Do you have a plan to change a text format?
